doc/src/modules/index.rst

```diff
@@ -16,49 +16,49 @@ access any SymPy module, or use the list below:
 .. toctree::
    :maxdepth: 2
 
-   core.rst
+   algebras.rst
+   assumptions/index.rst
+   calculus/index.rst
+   categories.rst
+   codegen.rst
    combinatorics/index.rst
-   ntheory.rst
-   crypto.rst
    concrete.rst
+   core.rst
+   crypto.rst
+   diffgeom.rst
+   solvers/diophantine.rst
    discrete.rst
-   evalf.rst
-   codegen.rst
-   numeric-computation.rst
    functions/index.rst
    geometry/index.rst
    holonomic/index.rst
+   simplify/hyperexpand.rst
+   solvers/inequalities.rst
    integrals/integrals.rst
    liealgebras/index.rst
    logic.rst
    matrices/index.rst
+   ntheory.rst
+   numeric-computation.rst
+   evalf.rst
+   solvers/ode.rst
+   solvers/pde.rst
+   parsing.rst
+   physics/index.rst
+   plotting.rst
    polys/index.rst
    printing.rst
-   plotting.rst
-   assumptions/index.rst
-   rewriting.rst
    series/index.rst
    sets.rst
    simplify/simplify.rst
-   simplify/hyperexpand.rst
-   stats.rst
-   solvers/ode.rst
-   solvers/pde.rst
    solvers/solvers.rst
-   solvers/diophantine.rst
-   solvers/inequalities.rst
    solvers/solveset.rst
```

Comment:
These could all be under a solvers header. Perhaps solvers.rst could be the main header there

Reply:
so i place all other files code in solvers.rst?

Reply:
No. I mean that there could be a header for solvers with the other things underneath like:
```
   * solvers
        * diophantine
        * inequalities
        ...
```
I'm not sure how to do that in the RST though...

Reply:
okay let me try

Reply:
done added all the solvers file in one heading and the rewriting.rst and evalf.rst to core